package.json

```diff
@@ -91,12 +91,13 @@
     "nodemon": "^2.0.14",
     "prettier": "^2.4.1",
     "react-test-renderer": "^17.0.2",
-    "snyk": "^1.745.0",
+    "snyk": "^1.749.0",
     "stmux": "^1.8.1",
     "ts-jest": "^27.0.7",
     "typescript": "^4.4.4",
     "wait-on": "^6.0.0",
-    "webpack": "^5.59.1",
+    "web-vitals": "^2.1.2",
```

Comment:
instead of adding this as a dependency, could we add it as a peerDependency instead?